### PR TITLE
Follow users

### PR DESCRIPTION
**Pull request by thiennv57**
Finish follow user

### CODE REVIEW

File: app/views/users/_user.html.erb [hunk 1]
```diff
@@ -1,3 +1,7 @@
 <li>
   <%= link_to user.name, user %>
+  <% if current_user.admin? && !current_user? user %>
```
Comment (nhs3108) [inline app/views/users/_user.html.erb]: I18n

File: app/views/users/_user.html.erb [hunk 2]
```diff
@@ -1,3 +1,7 @@
...
+    | <%= link_to "delete", user, method: :delete,
+                                  data: {confirm: "You sure?"} %>
```
Comment (nhs3108) [inline app/views/users/_user.html.erb]: I18n

Comment (nhs3108) [inline app/views/users/_user.html.erb]: I18n

File: app/views/users/_unfollow.html.erb [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<%= form_for(current_user.active_relationships.find_by(followed_id: @user.id),
```
Comment (phamkykhoi) [inline app/views/users/_unfollow.html.erb]: Remove  ( )

File: app/views/users/_follow_form.html.erb [hunk 4]
```diff
@@ -0,0 +1,9 @@
+<% unless current_user? @user %>
+  <div id="follow_form">
+    <% if current_user.following? @user %>
+      <%= render "unfollow" %>
```
Comment (phamkykhoi) [inline app/views/users/_follow_form.html.erb]: Move unfollow to Settings

File: app/views/users/_follow_form.html.erb [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+    <% else %>
+      <%= render "follow" %>
```
Comment (phamkykhoi) [inline app/views/users/_follow_form.html.erb]: Move follow to Settings

File: app/views/users/_unfollow.html.erb [hunk 6]
```diff
@@ -0,0 +1,6 @@
+<%= form_for current_user.active_relationships.find_by(followed_id: @user.id),
```
Comment (phamkykhoi) [inline app/views/users/_unfollow.html.erb]: Remove ( )

File: app/controllers/users_controller.rb [hunk 7]
```diff
@@ -1,6 +1,7 @@
 class UsersController < ApplicationController
   before_action :set_user, only: [:show, :edit, :update, :destroy]
-
+  before_action :logged_in_user, only: [:index, :edit, :update, :destroy, :following, :followers]
+  before_action :correct_user,   only: [:edit, :update]
   def index
```
Comment (dieunb) [inline app/controllers/users_controller.rb]: add new white line above

**Pull request merged**